src/client/containers/404Page/404Page.container.js

```diff
@@ -21,7 +22,9 @@ function Error404Page() {
         </div>
 
         <div>
-          <Buttons label="Go Home" size="big" isMono={true} />
+          <Link to="/">
+            <Buttons label="Go Home" size="big" isMono={true} />
+          </Link>
```

Comment:
This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an <a> tag around a <button> tag.

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

If you want to navigate to a page using a button this is the correct way to do so :).

https://reactrouter.com/web/api/Hooks/usehistory